client_taxi_driver/shared/src/commonMain/kotlin/domain/entity/Order.kt

```diff
@@ -0,0 +1,8 @@
+package domain.entity
+
+data class Order(
+    val id: String,
+    val passengerName: String,
+    val pickUpAddress: String,
+    val dropOffAddress:String,
```

Comment:
is this represent the name of the place or the coordination of the place ?

Reply:
The name of the place and i refactored it